### ISSUE TITLE
Ignore instead of throwing when encountering non-4326 SRIDs

### ISSUE THREAD

**Issue by mattbretl** [graphile/postgis#4]
The decision to only expose columns declared as SRID 4326 makes sense; automatic spatial transforms should not be an out-of-the-box feature. But I don't think it should throw upon encountering non-4326 columns. I wouldn't even log a warning.. I would just make it clear in the README that only columns with SRID 4326 will be exposed and all others will be ignored.

Background: Every PostGIS app I've worked on has stored some data in non-4326 coords. For example, original survey data (in a local Cartesian coordinate system) is loaded into PG to preserve the source of truth, and the transform is done via trigger functions or materialized views.

**Comment by benjie** [comment]
Actually I think we should just expose it all, and maybe output the srid. We can do this using the GeoJSON functions in more verbose mode. I need to understand more PostGIS first though

**Comment by mattbretl** [comment]
True, I was thinking in terms of web mapping where 4326 is by far the most useful, but as long as the SRID is documented/included, there's no reason to hide the other columns.

A few options for the SRID:
1. Add it to the GraphQL type name, e.g. `GeometryPoint3857`. Most apps only use a handful of SRIDs, so this shouldn't cause an explosion of types. The would provide the best documentation, with no need to query anything.
2. Add an `srid` field to the `Geometry*` types. Helpful for mixed-SRID columns (yikes), but if there's a SRID constraint in place, it's redundant to put the info on every node.
3. Expose the `crs` prop on GeoJSON via [ST_AsGeoJSON options](https://postgis.net/docs/ST_AsGeoJSON.html). Similar to (2), it's redundant info if there's a SRID constraint in place, but it's nice to have it merged into the dataset itself.

**Comment by mattbretl** [comment]
Just thought of another one:
4. Add it to the GraphQL field description. Probably preferable to (1).

**Comment by mattbretl** [comment]
I'm leaning toward:
- 4326: (4)
- all others: (3) and (4)

**Comment by benjie** [comment]
I like 4 and 1 (which depends on 3 anyway); handle it in the inflector? Might be easier to manage fewer types.
